src/content/index.tsx

```diff
@@ -8,7 +8,11 @@ export const moduleNames = {
     [Module.A_B_C_AB_AC]: "Competitive Binding",
 };
 
-export const FIRST_PAGE = 1;
+export const FIRST_PAGE = {
+    [Module.A_B_AB]: 0, // landing page
+    [Module.A_C_AC]: 1,
+    [Module.A_B_C_AB_AC]: 1,
+};
```

Comment:
could/should this be expressed in terms of the `Section` type? (e.g. `[Module.A_B_AB]: Section.LandingPage,`?) Or does that capture a different concept?